app/styles/_mobile-service-instance-list.less

```diff
@@ -5,12 +5,43 @@
   }
 
   .mobile-service-instance-row {
+    &.no-bindings {
+      pointer-events: none;
+      background-color: inherit;
+      border-color: @list-pf-border-color;
+      border-left-color: @color-pf-white;
+      border-right-color: @color-pf-white;
+      border-style: solid;
+      border-width: 1px;
+      border-bottom: none;
+
+      .list-pf-name {
+        .service-name {
+          pointer-events: all;
+        }
+      }
+      .list-pf-container {
+        cursor: default;
+      }
+      .list-pf-content {
+        flex-grow: 1;
+      }
+      .create-binding {
```

Comment:
Do these need to be in alphabetical order?

Reply:
Just the css has to alphabetical not the selectors.